types/expo/expo-tests.tsx

```diff
@@ -33,24 +30,29 @@ import {
     ImagePicker,
     IntentLauncherAndroid,
     KeepAwake,
-    KmlMapEvent,
     LinearGradient,
     Linking,
     Location,
     MailComposer,
-    MapEvent,
-    MapStyleElement,
     MapView,
     MediaLibrary,
     Permissions,
     PublisherBanner,
-    Region,
     registerRootComponent,
     ScreenOrientation,
     SecureStore,
     Svg,
     Updates
 } from 'expo';
+import {
```

Comment:
Sorry for asking all these questions, but why are these properties moved to `react-native-maps`? It's my understanding that `expo` contains everything that is needed, so I would not have include `react-native-maps` into my `package.json`.

Reply:
I moved them because there I had some error, but just moved them back. The expo definitions just re-export stuff from `react-native-maps`, so it's the same.

Reply:
👍